README.md

```diff
@@ -71,12 +79,5 @@ __IMPORTANT NOTICE:__ The contents of this repository currectly reflect a __DRAF
 1. Customization
    1. [Custom Events](./customization/custom-events.md)
    1. [Custom Data](./customization/custom-data.md)
-1. Implementations
-   1. [Event Persistence](./implementations/event-persistence.md)
-   1. [Event Aggregation and Analysis](./implementations/event-aggregation-and-analysis.md)
-   1. Activity Orchestration
-   1. Event Transport and Routing
-   1. [Event Dispatch](./implementations/event-dispatch.md)
-   1. [Visualization](./implementations/visualization.md)
 1. Extensions
    1. [Eiffel Operations Extension](https://github.com/Ericsson/eiffel-operations-extension)
```

Comment:
This is probably the only (?) reference to old Ericsson labeled stuff in this repository. I understand it is not feasible to move that to eiffel-community within this PR, but maybe the link should have an accompanying comment stating that it is planned to be moved to eiffel-community in due time?

Reply:
I disagree. Sure, we can move the operations extension to eiffel-community if those involved in it wish to do so (although that would actual active maintainers). That's beside the point, though - extensions could be provided by John Doe and his second cousins and be published wherever for all I care. This one just happens to be hosted by the Ericsson organization.

Does that make sense?

Reply:
Accepted